### PR TITLE
Website rework

### PR DESCRIPTION
**Pull request by ConsoleTVs**
I reworked the official site with some simple styling, keeping it fast and minimal,
while giving some styling to it to make it more visually appealing.

## Features

I focused on the following:

- No JS (there's no need for it for a simple landing page)
- Low bundle size (~ 5KB of CSS, no need to minify the HTML or styles.css)
- Prefetch DNS for the google fonts.
- Utility-first CSS
- Prettier (to have a consisten code style, I hope there's no issue by leaving the config file there)

## Image

image

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 2
```
Comment (ConsoleTVs) [inline .prettierrc:2]: This config file may be removed, but keeping it here will allow consistent code style.

File: index.html [hunk 2]
```diff
@@ -1,16 +1,117 @@
...
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>The Entropic Package Manager</title>
+    <!-- Tailwind CSS -->
+    <link href="tailwind.css" rel="stylesheet" />
+    <!-- Specific styles -->
+    <link rel="stylesheet" href="styles.css" />
+    <!-- Fonts -->
+    <link rel="dns-prefetch" href="//fonts.gstatic.com" />
```
Comment (ConsoleTVs) [inline index.html:13]: DNS prefetch for the google fonts (for performance)

File: index.html [hunk 3]
```diff
@@ -1,16 +1,117 @@
...
+        </h4>
+        <a href="https://github.com/entropic-dev/entropic">
+          <svg
+            class="mt-4 w-8 h-8 fill-current text-white"
+            role="img"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <title>GitHub icon</title>
+            <path
```
Comment (ConsoleTVs) [inline index.html:38]: Inline SVG instead of an image for performance reasons.

File: index.html [hunk 4]
```diff
@@ -1,16 +1,117 @@
...
+            />
+          </svg>
+        </a>
+      </div>
+    </div>
+    <div class="container mx-auto -mt-24 px-4 md:px-0">
+      <div class="rounded bg-white shadow-lg p-12 leading-relaxed text-lg mb-6">
+        <h1 class="text-2xl mb-6">What is Entropic?</h1>
+        <p>
+          A new package registry with a new CLI, designed to be easy to stand up
```
Comment (ConsoleTVs) [inline index.html:49]: Copy-paste from the readme of the github page.

**Comment by chrisdickinson** [comment]
Hey! Thanks so much, & sorry for the review delay. This looks great! I'm merging it now.

**Comment by ConsoleTVs** [comment]
Thanks a lot @chrisdickinson ! I'm happy you liked it :)

**Pull request merged**